### ISSUE TITLE
priests aren't strong enough

### ISSUE THREAD

**Issue by boykevink** [Unlimited-WoW/BugTracker#11]
so as a fully geared rogue and a fully geared warlock in battleground i let a priest put 2 dots on me.
so my warlock dots dont do near as much damage as 2 dots from a priest.
the priest can kill me with just 2 dots but i can spam all my warlock dots on him and cast shadow bolts,chaos bolts and soul fire on to do no damage on a priest at all when he kills me with just the 2 dots.
oh yea they kill warlocks minion like its a lvl1 mob i gues warlocks minion's are usefull for other things huh
as a rogue now they just  kill you simply with 2 dots and thats it.
i belive priest needs to be buffed more.i mean they should be able to kill with just 1 dot on a rogue or warlock.
why should they need to cast 2 dots. i mean thats way toomuch of an effort for them.
so basically just give priest more spellpower and we should all be happy on the server since they need 2 dots to kill.
